api/__tests__/get.test.js

```diff
@@ -95,49 +94,87 @@ describe("GET /api/topics", () => {
       });
   });
 
-  describe("/api/articles", () => {
-    test(`to GET status 200 and an array containing article objects containing the following properties: author, title, article_id, body, topic, created_at, votes, article_img_url, comment_count. The articles should be sorted by date in descending order.`, () => {
+  test("returns articles sorted by date descending", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        expect(articles).toBeSorted({
+          key: "created_at",
+          descending: true,
+        });
+      });
+  });
+  describe("GET /api/articles/:article_id/comments", () => {
+    test(`returns status 200 and an array containing comment objects containing the following properties: comment_id, votes, created_at, author, body, article_id. Comments should be sorted by date in descending order.`, () => {
       return request(app)
-        .get("/api/articles")
+        .get("/api/articles/1/comments")
         .expect(200)
         .then((response) => {
-          const articles = response.body.articles;
-
+          const comments = response.body.comments;
           const expectedProperties = {
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
             author: expect.any(String),
-            title: expect.any(String),
+            body: expect.any(String),
             article_id: expect.any(Number),
```

Comment:
you know what the article_id is because of the endpoint, so it would be good to include that here :)